grounding.py

```diff
@@ -103,6 +104,6 @@ def ground(self, imagefile, segments, _=None):
         classes = list(classes)
 
         classes = classes_to_numpy(classes)
-        print "classified ", numpy.count_nonzero(classes != classes_to_numpy(BLANK_CLASS)), "characters out of", max(
-            classes.shape)
+        print(("classified ", numpy.count_nonzero(classes != classes_to_numpy(BLANK_CLASS)), "characters out of", max(
```

Comment:
That's a `2to3` automatic change, I ran it again on this fule for TermonalGrounder.